plugin/pkg/scheduler/scheduler.go

```diff
@@ -222,7 +225,7 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 	if err != nil {
```

Comment:
Give me some time to check history.

That's error-prone part, we used to close some PRs which want to enhance it. Another concern is that our UT and e2e did not catch it :(.